### PR TITLE
FEATURE: New content

### LINKED ISSUE #147: New Demo for Neos 8.3
## Ideas from Neos Sprint Hamburg 2022

- [x] Presentation folder with all components inside
  - [x] More complex components like columns will have a subfolder to contain their parts
  - [x] Remove atoms, section and block folders
- [x] Document Mixin in Page mergen
- [x] Remove restrictions and enforce adding constraints on all nodetypes @Sebobo 
- [x] Rename CKConfig 
- [ ] Add readme to subfolders to explain their use
- [x] MenuItems Recursion @jonnitto 
- [x] Remove context dependent setting and replace with simple setting access set by context
- [x] Rename Splide to Slider
- [ ] Release notes should contain information about pruning and reimporting the new content
- [ ] Explain that Tailwind is an example and people could use anything
- [x] Chapters still have to be renamed @jonnitto 

## v9 new CR specifics

- [x] Create separate issue #156

### PR DESCRIPTION
**Pull request by Sebobo**
Depends on #137

### CODE REVIEW

**Review by regniets — APPROVED** [review]
Checked through your changes - Kudos, really like them.
Concerning demo.neos.io, i am ok with merging. 
I really like the code style editor plugin!
Not quite sure about the rest - what else is the package being used for?

**Comment by ahaeslich** [comment]
@jonnitto, @Sebobo  Do we have an overview what still needs to be done here? And are we aiming to include the refactoring for the 8.3 release?

**Comment by Sebobo** [comment]
I really want this for 8.3 as it would also be nice for the NeosCon and the contribution day.

File: Resources/Private/Fusion/Root.fusion [hunk 1]
```diff
@@ -1,4 +1,7 @@
 ##
 # Include all .fusion files
 #
-include: ./**/*.fusion
+include: **/*.fusion
+// TODO Remove this line when the following PR is merged:
+// https://github.com/neos/neos-development-collection/pull/3903
+include: ../../../NodeTypes/**/*.fusion
```
Comment (ahaeslich) [inline Resources/Private/Fusion/Root.fusion]: The pull request was merged. We should change this line.
It should work with `include: nodetypes://Neos.Demo/**/*.fusion`

**Comment by Sebobo** [comment]
@ahaeslich @mficzel are we done here now? 

We might need to update the content export with the blog pages so we don't have inconsistencies, but is there anything else?

**Comment by mficzel** [comment]
Content for the blog would be great and I think we wanted to add some BlogList content element

**Comment by mficzel** [comment]
I pushed a new commit wich adds `Content.BlogPostingList`and introduces `Cards.Container` and `Cards.Cart` on the presentation side`

Both are still ugly even though i tried to use the card components from https://tailwind-elements.com/docs/standard/components/cards/ .. so someone with little tailwind knowhow and a non-backend eye should adjust those.

**Comment by ahaeslich** [comment]
> @ahaeslich @mficzel are we done here now?
> 
> We might need to update the content export with the blog pages so we don't have inconsistencies, but is there anything else?

Done ✅

**Comment by ahaeslich** [comment]
@Sebobo & @mficzel I think we can merge this now and add all improvements in smaller merge requests

I would check and add #158 afterwards.

**Review by ahaeslich — APPROVED** [review]

**Review by mficzel — APPROVED** [review]
Agree to get this in now and fix remaining issues later. Actually like the state but would like to know what @Sebobo and @jonnitto say.

**Comment by mhsdesign** [comment]
Happy to see that we are on the final track ;)

i guess Jon and Seb will have a closer look at the demo on the neos to 100 day and if they find something we can still correct it then ;)

**Comment by mficzel** [comment]
Yes I would not wait for feedback but give them some days.

**Comment by mficzel** [comment]
Hmm ... this change breaks functional tests in the neos-development collection because of dropping the .html suffix. Do we want this that much to adjust the tests or do we want to revert that part.

```
2) Neos\Neos\Tests\Functional\Service\LinkingServiceTest::linkingServiceCreatesUriViaGivenNodeObject
Failed asserting that '/en/home' ends with "en/home.html".

/var/www/html/Packages/Neos/Neos.Neos/Tests/Functional/Service/LinkingServiceTest.php:336
/var/www/html/Packages/Neos/Neos.Neos/Tests/Functional/Service/LinkingServiceTest.php:149
/var/www/html/bin/phpunit:123

3) Neos\Neos\Tests\Functional\Service\LinkingServiceTest::linkingServiceCreatesUriViaAbsoluteNodePathString
Failed asserting that '/en/home' ends with "en/home.html".

/var/www/html/Packages/Neos/Neos.Neos/Tests/Functional/Service/LinkingServiceTest.php:336
/var/www/html/Packages/Neos/Neos.Neos/Tests/Functional/Service/LinkingServiceTest.php:157
/var/www/html/bin/phpunit:123

4) Neos\Neos\Tests\Functional\Service\LinkingServiceTest::linkingServiceCreatesUriViaStringStartingWithTilde
Failed asserting that '/en/home' ends with "en/home.html".

.... and so on

```

**Comment by mhsdesign** [comment]
@mficzel  i think we should fix the tests instead of reintroducing the suffix ^^ (discussed with @ahaeslich)

**Pull request merged**